packages/headless/src/features/facets/range-facets/date-facet-set/date-facet-actions.ts

```diff
@@ -116,6 +116,11 @@ export function validateManualDateRanges(
 
   options.currentValues.forEach((value) => {
     const {start, end} = buildDateRange(value);
+    if (dayjs(start).isSame(dayjs(), 'day') && dayjs(end).isValid) {
+      throw new Error(
+        `The date range with end value ${value.end} has no start value`
```

Comment:
The problem with this addition is it is no longer possible to create a range where the `start` is set to the current day.

My understanding of the jira is when a `start` is not specified, the current date is used by default. Instead of using a default, we want to alert the developer to specify a `start`, which would be more helpful than the headless error message. The change would be in Atomic rather than headless.

Reply:
Ok, I will try to approach it from this angle instead.